### PR TITLE
Upgrade to ruby 3.1

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -136,7 +138,8 @@ DEPENDENCIES
   sinatra
   sinatra-contrib
   timecop
+  webrick (~> 1.7)
```
Comment (ErikSchierboom) [inline Gemfile.lock:141]: Without this, I get:

```console
~/exercism/tooling-orchestrator$ bundle exec rake test

File does not exist: webrick/httputils

rake aborted!
Command failed with status (1)
/usr/share/rvm/gems/ruby-3.1.0/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
/usr/share/rvm/gems/ruby-3.1.0/bin/ruby_executable_hooks:22:in `eval'
/usr/share/rvm/gems/ruby-3.1.0/bin/ruby_executable_hooks:22:in `<main>'
Tasks: TOP => test
(See full trace by running task with --trace)
```

File: .github/workflows/rubocop.yml [hunk 2]
```diff
@@ -14,16 +14,20 @@ jobs:
...
-          ruby-version: 2.6.6
+          ruby-version: .ruby-version
 
       - name: Install gems
         run: |
-          gem install rubocop -v 0.91.0
-          gem install rubocop-minitest
-          gem install rubocop-performance
-          gem install rubocop-rails
+          # Extract the exact versions of the rubocop gems from Gemfile.lock
```
Comment (ErikSchierboom) [inline .github/workflows/rubocop.yml:23]: I copied this from the website repo

**Comment by iHiD** [comment]
I'm happy for us to merge this one.

**Comment by ErikSchierboom** [comment]
Superseded by https://github.com/exercism/tooling-orchestrator/pull/49

**Pull request closed without merging**